pkg/podmounter/mppod/creator.go

```diff
@@ -24,9 +25,10 @@ type ContainerConfig struct {
 
 // A Config represents configuration for spawned Mountpoint Pods.
 type Config struct {
-	Namespace string
-	Version   string
-	Container ContainerConfig
+	Namespace        string
+	Version          string
```

Comment:
Same with the label, could you rename it to `MountpointVersion`?

Reply:
Done!